Description: fix ftbfs with gcc 10 due to missing C++ header include
Author: Steve Langasek <steve.langasek@ubuntu.com>
Last-Update: 2020-08-03
Applied-Upstream: https://github.com/rspamd/rspamd/commit/06c6c939ae59516a720f0cfc5a29e214b6569476.patch
Origin: upstream
Bug: https://bugs.debian.org/966475
Forwarded: not-needed
Reviewed-By: Sebastien Badia <sbadia@debian.org>

Index: rspamd-2.5/contrib/replxx/src/io.cxx
===================================================================
--- rspamd-2.5.orig/contrib/replxx/src/io.cxx
+++ rspamd-2.5/contrib/replxx/src/io.cxx
@@ -3,6 +3,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <array>
+#include <stdexcept>
 
 #ifdef _WIN32
 
